day_4/task_1/main.js

```diff
@@ -0,0 +1,22 @@
+const buttons = document.querySelectorAll('button')
+const result = document.getElementById('result')
+let resultValue = '';
```

Comment:
lets add a blank space after variable declarations, conditions, loops etc for code readability.